expe.py

```diff
@@ -0,0 +1,5 @@
+#This defines the structure of the experiment
+
+
+class Expe(object):
```

Comment:
Prefer complete words for class names -- it makes the code easier to read for others, and future you.